### PR TITLE
Set $supportedMediaTypes to application/json

### PR DESCRIPTION
**Pull request by kdambekalns**
Fixes https://github.com/t3n/graphql/issues/29

### CODE REVIEW

File: Classes/Controller/GraphQLController.php [hunk 1]
```diff
@@ -51,6 +51,14 @@ class GraphQLController extends ActionController
      */
     protected $requestLogger;
 
+    /**
+     * A list of IANA media types which are supported by this controller
+     *
+     * @var array
```
Comment (johannessteu) [inline Classes/Controller/GraphQLController.php:57]:
```suggestion
     * @var string[]
```

File: Classes/Controller/GraphQLController.php [hunk 2]
```diff
@@ -51,6 +51,14 @@ class GraphQLController extends ActionController
      */
     protected $requestLogger;
 
+    /**
+     * A list of IANA media types which are supported by this controller
+     *
+     * @var string[]
+     * @see http://www.iana.org/assignments/media-types/index.html
```
Comment (johannessteu) [inline Classes/Controller/GraphQLController.php:58]:
```suggestion
     * @var string[]
     *
     * @see http://www.iana.org/assignments/media-types/index.html
```

File: Classes/Controller/GraphQLController.php [hunk 3]
```diff
@@ -51,6 +51,15 @@ class GraphQLController extends ActionController
      */
     protected $requestLogger;
 
+    /**
+     * A list of IANA media types which are supported by this controller
+     *
+     * @var string[]
+     *
+     * @see http://www.iana.org/assignments/media-types/index.html
```
Comment (johannessteu) [inline Classes/Controller/GraphQLController.php:59]:
```suggestion
     * @see http://www.iana.org/assignments/media-types/index.html
     *
     * @var string[]
```

**Comment by kdambekalns** [comment]
Lovely… if that linter would explain _what_ is wrong… 😎 Thanks for fixing it!

**Comment by johannessteu** [comment]
Yeah >.>

**Pull request merged**